### ISSUE TITLE
`npm run install` is detected as DL3016

### ISSUE THREAD

**Issue by fvj** [hadolint/hadolint#256]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
`npm run install` should not be detected as a violation of DL3016.

### Actual behavior
`npm run install` is a violation of DL3016.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.11.0-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
FROM scratch

RUN npm run install
```
results in 
```Dockerfile:3 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>```
